src/legacy/server/saved_objects/service/lib/errors.test.ts

```diff
@@ -354,6 +354,7 @@ describe('savedObjectsClient/errorTypes', () => {
     describe('createEsAutoCreateIndexError', () => {
       it('does not take an error argument', () => {
```

Comment:
This is a bit of a weird test for me. Typescript complains because we supply arguments which don't exist, so I added a ts-ignore for now, but was wondering if this test is really asserting anything useful, perhaps we should just delete the whole test.